include/drivers/spi.h

```diff
@@ -124,22 +124,24 @@ extern "C" {
  * This can be used to control a CS line via a GPIO line, instead of
  * using the controller inner CS logic.
  *
- * @param gpio_dev is a valid pointer to an actual GPIO device. A NULL pointer
- *        can be provided to full inhibit CS control if necessary.
- * @param gpio_pin is a number representing the gpio PIN that will be used
- *    to act as a CS line
- * @param delay is a delay in microseconds to wait before starting the
- *    transmission and before releasing the CS line
- * @param gpio_dt_flags is the devicetree flags corresponding to how the CS
- *    line should be driven. GPIO_ACTIVE_LOW/GPIO_ACTIVE_HIGH should be
+ * @param gpio devicetree specification of CS GPIO.
+ *    The device pointer can be set to NULL to fully inhibit CS control if
+ *    necessary. The GPIO flags GPIO_ACTIVE_LOW/GPIO_ACTIVE_HIGH should be
  *    equivalent to SPI_CS_ACTIVE_HIGH/SPI_CS_ACTIVE_LOW options in struct
  *    spi_config.
+ * @param delay is a delay in microseconds to wait before starting the
+ *    transmission and before releasing the CS line
```

Comment:
`@param` is not the proper way of documenting structs (was wrong), please move to
```c
struct x {
    /** Documentation for foo. */
    int foo;
};
```

Reply:
Fixed.